config/schemas/config.yaml

```
@@ -127,6 +371,1362 @@ properties:
+                pattern:
+                  default: other-*
+                  type: string
+                sizeGB:
+                  default: 1
+                  type: number
+              type: object
+            type: array
+          startingDeadlineSeconds:
+            default: 600
+            type: number
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+        type: object
+      dashboards:
+        additionalProperties: false
+        properties:
+          affinity:
+            $ref: '#/$defs/kubernetesAffinity'
+          nodeSelector:
+            $ref: '#/$defs/kubernetesNodeSelector'
+          resources:
+            $ref: '#/$defs/kubernetesResourceRequirements'
+          subdomain:
+            default: opensearch
+            type: string
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+          topologySpreadConstraints:
+            $ref: '#/$defs/kubernetesTopologySpreadConstraints'
+        type: object
+      dataNode:
+        additionalProperties: false
+        properties:
+          affinity:
+            $ref: '#/$defs/kubernetesAffinity'
+          count:
+            default: 2
+            type: number
+          dedicatedPods:
+            default: true
+            type: boolean
+          javaOpts:
+            default: -Xms512m -Xmx512m
+            type: string
+          nodeSelector:
+            $ref: '#/$defs/kubernetesNodeSelector'
+          resources:
+            $ref: '#/$defs/kubernetesResourceRequirements'
+          storageClass:
+            examples:
+              - '-'
+            type:
+              - string
+              - "null"
+            description: |
+              If null, no storageclass is specified in pvc and default storageClass is used
+              if the DefaultStorageClass admission plugin is enabled.
+              If "-", "" will be used as storageClassName.
+          storageSize:
+            default: 25Gi
+            type: string
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+        type: object
+      defaultTemplates:
+        default: true
+        type: boolean
+      enabled:
+        default: true
+        type: boolean
+      exporter:
+        additionalProperties: false
+        properties:
+          resources:
+            $ref: '#/$defs/kubernetesResourceRequirements'
+          serviceMonitor:
+            additionalProperties: false
+            properties:
+              interval:
+                default: 30s
+                type: string
+              scrapeTimeout:
+                default: 30s
+                type: string
+            type: object
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+        type: object
+      extraRoleMappings:
+        items:
+          additionalProperties: false
+          properties:
+            definition:
+              additionalProperties: false
+              properties:
```

`backend_roles` should be added here, and it would be great if `backend_roles` could be added and documented in the config as well as part of this PR as per [this issue](https://github.com/elastisys/compliantkubernetes-apps/issues/1148).
```suggestion
              properties:
                backend_roles:
                  items:
                    type: string
                  type: array
```

---

Adding `title`, `description` here would also be great and work with [the new `ck8s explain`](https://github.com/elastisys/compliantkubernetes-apps/pull/2112).